lib/aptible/auth/oauth2_config.rb

```diff
@@ -0,0 +1,5 @@
+require 'oauth2'
+
+OAuth2::Response.register_parser(:json, ['application/json', 'text/javascript', 'application/hal+json']) do |body|
```

Comment:
I'm not quite sure where the right place is to put this. It's strictly initialization code. Also, this code [already exists in the master branch of the oauth2 gem](https://github.com/oauth-xx/oauth2/blob/65749fc9bb76723d3274f30c6fde20cd3f2d7c5a/lib/oauth2/response.rb#L124).